src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -452,7 +453,7 @@ private void populateWritersFromLibraryParams() {
     private Map<String, String> buildSamHeaderParameters(final List<String> barcodes) {
         final Map<String, String> params = new LinkedHashMap<>();
 
-        String platformUnit = RUN_BARCODE + "." + LANE;
+        String platformUnit = RUN_BARCODE + "." + LANE.stream().map(Object::toString).collect(Collectors.joining(","));
```

Comment:
I wonder if it would be more correct to put each lane into a separate read group?  That's effectively what you'd get today if you ran per-lane and then merged the resulting BAMs.  Any idea how hard that would be to implement?